src/assets/App.css

```diff
@@ -0,0 +1,44 @@
+.App {
+  max-width: 1400px;
+  min-height: 1097px;
+  margin: 3rem auto;
+
+  /* margin: 0 40px 65px 0; */
+
+  /* padding: 0 0 794px; */
```

Comment:
- [ ] I recommend removing the commented code because they make your codebase unclean.